### PR TITLE
Query multiple tables

### PR DESCRIPTION
**Pull request by sntgopl**
# Request Review for Pull Request

### - `CREATE TABLE` for owners:
Screenshot 2023-05-06 at 10 39 38

### - `ALTER TABLE` for animals columns:
Screenshot 2023-05-06 at 10 46 41
Screenshot 2023-05-06 at 10 46 58

### - `INSERT INTO` owners data:
Screenshot 2023-05-06 at 10 53 27
Screenshot 2023-05-06 at 10 55 55

### - `INSERT INTO` species data:
Screenshot 2023-05-06 at 10 56 08

### - `UPDATE` data in animals regarding species:
Screenshot 2023-05-06 at 10 59 09

### - `UPDATE` data in animals regarding owners:
Screenshot 2023-05-06 at 11 03 25

### - `QUERY` What animals belong to Melody Pond?
Screenshot 2023-05-06 at 11 05 26

### - `QUERY` List of all animals that are Pokemon (their type is Pokemon).
Screenshot 2023-05-06 at 11 07 05

### - `QUERY` List all owners and their animals, remember to include those that don't own any animal:
Screenshot 2023-05-06 at 11 10 40

### - `QUERY` How many animals are there per species?
Screenshot 2023-05-06 at 11 13 42

### - `QUERY` List all Digimon owned by Jennifer Orwell.
Screenshot 2023-05-06 at 11 16 39

### - `QUERY` List all animals owned by Dean Winchester that haven't tried to escape.
Screenshot 2023-05-06 at 11 20 43

### - `QUERY` Who owns the most animals?
Screenshot 2023-05-06 at 11 25 44

## Please,
Let me know if there's anything else I should add to my project. 

Thank you

Regards,
Santiago.

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -12,4 +12,30 @@ INSERT INTO animals VALUES (7, 'Squirtle', '1993-04-02', 3, false, -12.13);
...
+
+INSERT INTO owners (full_name, age)
+    VALUES 
+        ('Sam Smith', 34),
+        ('Jennifer Orwell', 19),
+        ('Bob', 45),
+        ('Melody Pond', 77),
+        ('Dean Winchester', 14),
+        ('Jodie Whittaker', 38);
+
```
Comment (gemmen29) [inline data.sql:25]: - [ ] Kindly add the statement of inserting `Pokemon` and `Digimon` into the species table in your `data.sql` file as per this [requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_multiple_tables.md#:~:text=Insert%20the%20following%20data%20into%20the%20species,Digimon).

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @sntgopl, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 💯 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: data.sql (hunk 1 above)
Comment (sntgopl) [reply data.sql:25]: Hello @gemmen29, thank you for your time and attention. I've missed that step, and have already added it. Again thank you for your review.

**Review by Qoosim — APPROVED** [review]
#  *Status: Approved* :tada: :heavy_check_mark: 

Hi @sntgopl, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

## Highlights
-  Good README file. :heavy_check_mark: 
-  Correct Gitflow. :heavy_check_mark: 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by sntgopl** [comment]
Thank you for your time!

**Pull request merged**